main/lsp/ShowOperation.cc

```diff
@@ -8,16 +8,58 @@ namespace sorbet::realmain::lsp {
 using namespace std;
 
 namespace {
+
 unique_ptr<LSPMessage> makeShowOperation(std::string operationName, std::string description,
                                          SorbetOperationStatus status) {
     return make_unique<LSPMessage>(make_unique<NotificationMessage>(
         "2.0", LSPMethod::SorbetShowOperation,
         make_unique<SorbetShowOperationParams>(move(operationName), move(description), status)));
 }
+
+string_view kindToOperationName(ShowOperation::Kind kind) {
+    switch (kind) {
+        case ShowOperation::Kind::Indexing:
+            return "Indexing";
+            break;
+        case ShowOperation::Kind::SlowPathBlocking:
+            return "SlowPathBlocking";
+            break;
+        case ShowOperation::Kind::SlowPathNonBlocking:
+            return "SlowPathNonBlocking";
+            break;
+        case ShowOperation::Kind::References:
+            return "References";
+            break;
+        case ShowOperation::Kind::SymbolSearch:
+            return "SymbolSearch";
+            break;
+    }
+}
+
+string_view kindToDescription(ShowOperation::Kind kind) {
+    switch (kind) {
+        case ShowOperation::Kind::Indexing:
+            return "Indexing files...";
+            break;
+        case ShowOperation::Kind::SlowPathBlocking:
+            return "Typechecking...";
+            break;
+        case ShowOperation::Kind::SlowPathNonBlocking:
+            return "Typchecking in background";
+            break;
+        case ShowOperation::Kind::References:
+            return "Finding all references...";
+            break;
+        case ShowOperation::Kind::SymbolSearch:
+            return "Workspace symbol search...";
+            break;
```

Comment:
```suggestion
            return "Indexing files...";
        case ShowOperation::Kind::SlowPathBlocking:
            return "Typechecking...";
        case ShowOperation::Kind::SlowPathNonBlocking:
            return "Typchecking in background";
        case ShowOperation::Kind::References:
            return "Finding all references...";
        case ShowOperation::Kind::SymbolSearch:
            return "Workspace symbol search...";
```

Reply:
oops, thanks

that was copy paste from when i did this in one `switch` statement in the constructor, but that would have required me to make the fields non-const in order to write into `this->operationName` etc.